client/src/SessionsList.js

```diff
@@ -9,7 +9,8 @@ import {
   ListSubheader,
 } from '@material-ui/core'
 import { makeStyles } from '@material-ui/core/styles'
-import { Add, Done } from '@material-ui/icons'
+import { Add } from '@material-ui/icons'
+import EditOutlinedIcon from '@material-ui/icons/EditOutlined'
```

Comment:
please import the Icon by destructuring (as above in line 12), to avoid multiple `@material-ui/icons` imports